mappings/net/minecraft/network/PacketByteBuf.mapping

```diff
@@ -51,7 +51,7 @@ CLASS net/minecraft/class_2540 net/minecraft/network/PacketByteBuf
 	COMMENT  <td>{@link UUID}</td><td>{@link #readUuid()}</td><td>{@link #writeUuid(UUID)}</td>
 	COMMENT </tr>
 	COMMENT <tr>
-	COMMENT  <td>{@link NbtCompound}</td><td>{@link #readCompound()}</td><td>{@link #writeCompound(NbtCompound)}</td>
+	COMMENT  <td>{@link NbtCompound}</td><td>{@link #readNbt()}</td><td>{@link #writeNbt(NbtCompound)}</td>
```

Comment:
What? The API's public type is `@Nullable NbtCompound`.

Reply:
When used as a root element the `Compound` part can be omitted, else it would need to be renamed to readNbtCompound, etc. which I'm not fond off and you would need to also redo methods in NbtIo.

Reply:
Sure, go for this rename. This rename is consistent and logical.